.readthedocs.yml

```diff
@@ -19,4 +19,3 @@ python:
       - requirements: doc/dev_requirements.txt
       - method: setuptools
         path: .
-    system_packages: true
```

Comment:
the need (and support) for this [has been dropped](https://blog.readthedocs.com/drop-support-system-packages/)